templates/artist/artist_detail.html

```diff
@@ -200,6 +226,69 @@ <h1 class="header-font">{{ artist.name }}</h1>
 {% block extrajs %}
     <script src="https://checkout.stripe.com/checkout.js"></script>
     <script type="text/javascript">
```

Comment:
Since you created a new open `<script>` tag, remove this one that was left behind.